### PR TITLE
Fixed SQL Client naming scheme.

### PR DESCRIPTION
**Pull request by aznashwan**

### CODE REVIEW

File: management/sql/client.go [hunk 1]
```diff
@@ -25,22 +25,22 @@ const (
 	DatabaseStateCreating = "Creating"
 )
 
-// SqlDatabaseClient defines various database CRUD operations.
+// Client defines various database CRUD operations.
```
Comment (ahmetalpbalkan) [inline management/sql/client.go]: This should be SQLClient just to be consistent with other packages.

Comment (ahmetalpbalkan) [inline management/sql/client.go]: Or SQLDatabaseClient

Comment (aznashwan) [inline management/sql/client.go]: Will do.
However; I must warn you that [on my setup, at least], go lint cries that using it as sql.SQLDatabaseClient in other packages is needless repetition; that's why I made it into just Client in the first place...

**Comment by aznashwan** [comment]
@ahmetalpbalkan fixed; and the CI seems happy now :smile:

**Pull request merged**